ingestion/functions/parsing/argentina/argentina.py

```diff
@@ -0,0 +1,233 @@
+import os
+import sys
+from datetime import datetime
+import csv
+
+# Layer code, like parsing_lib, is added to the path by AWS.
+# To test locally (e.g. via pytest), we have to modify sys.path.
+# pylint: disable=import-error
+try:
+    import parsing_lib
+except ImportError:
+    sys.path.append(
+        os.path.join(
+            os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
+            'common/python'))
+    import parsing_lib
+
+
+import os
+import sys
+from datetime import date, datetime
+import csv
+
+
+private_public_map = {'Público': 'Public', 'Privado': 'Private'}
+
+
+def convert_date(date_str: str):
+    """
+    Convert raw date field into a value interpretable by the dataserver.
+
+    The date is listed in YYYY-mm-dd format
+    """
+    date = datetime.strptime(date_str, "%Y-%m-%d")
+
+    return date.strftime("%m/%d/%Y")
+
+
+def convert_gender(raw_gender):
+    if raw_gender == "M":
+        return "Male"
+    if raw_gender == "F":
+        return "Female"
+
+
+def convert_location(entry):
+    '''
+    The only information we have is the province where case was diagnosed/hospitalised
+    '''
+    if entry['carga_provincia_nombre']:
+        if entry['carga_provincia_nombre'] == 'CABA':
+            return {
+                "query": "Buenos Aires, Argentina",
+                "limitToCountry": "Argentina"}
+        else:
+
+            return {
+                "query": f"{entry['carga_provincia_nombre']}, Argentina",
+                "limitToCountry": "Argentina"}
```

Comment:
you can't specify limitToCountry yet, it hasn't been implemented https://github.com/globaldothealth/list/issues/1306

Last part of this issue should be to go through all the parsers and add that country restriction to them, you can ignore for now.

Reply:
ok will make new branch and remove limitToCountry, thanks

Reply:
not done?